api/tasks_test.go

```diff
@@ -364,7 +365,7 @@ func TestTask_Artifact(t *testing.T) {
 	if *a.GetterMode != "file" {
 		t.Errorf("expected file but found %q", *a.GetterMode)
 	}
-	if *a.RelativeDest != "local/foo.txt" {
+	if filepath.ToSlash(*a.RelativeDest) != "local/foo.txt" {
```

Comment:
shouldn't the `"local/foo.txt` also be wrapped in `filepath.ToSlash` for the equality to succeed?

Reply:
@preetapan Nope, `filepath.ToSlash` converts paths from a system dependent path to a unix style `/foo/bar` (https://godoc.org/path/filepath#ToSlash)